Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -2471,6 +2488,12 @@ private bool FlipBuffersForDeviceIfNecessary(InputDevice device, InputUpdateType
             return flipped;
         }
 
+        internal struct NoiseFilterElementState

```

Comment:
Didn't catch this before. This won't work without `[Serializable]`. So essentially, the current domain reload survival logic doesn't really do anything for noise filter :)

Recommend adding a test that makes sure we can successfully take noise filters across.